### DISCUSSION TITLE
why use different html tags with tailwind?

### DISCUSSION THREAD

**Discussion by fredbouin** [tailwindlabs/tailwindcss#3553] [General]
I've been using tailwind for my current project and it occurred to me that I don't really need any html tags other than div since I'm not styling them in a separate css file. Is there any point in me saying that my title is an h1 rather than simply wrapping it in a div? I'm thinking maybe in SEO google might look for specific tags? or maybe for accessibility for some reason I'm not aware of.

**Comment by lkraav** [comment]
Yes, tags semantics are a thing both for browsers, screen readers (impaired vision people) and SEO engines. Maybe more.

**Reply by fredbouin** [reply]
Right I figured as much, I think I saw someone writing tailwind code with mostly only divs, it seemed odd but I figured I'd ask, it would be nice to skip all those tags altogether!
